### PR TITLE
Added Professor Login

### LINKED ISSUE #474: Support instructor login
- [ ] When a professor or course coordinator logs in to the app, their user course for the course they teach should be set to professor

@willstenzel can you drop the log in schema returned by Khoury Admin here?

### PR DESCRIPTION
**Pull request by NEUDitao**
image

Closes #474

### CODE REVIEW

File: packages/server/src/seed/seed.controller.ts [hunk 1]
```diff
@@ -146,6 +146,20 @@ export class SeedController {
...
       });
+      // Professor (Snarky!!)
+      const user5 = await UserFactory.create({
+        email: 'li.edwa@northeastern.edu',
+        name: 'Eddy Li',
+        firstName: 'Eddy',
+        lastName: 'Li',
+        photoURL:
+          'https://ca.slack-edge.com/TE565NU79-UR6P32JBT-a6c89822c544-512',
+      });
```
Comment (dfarooq610) [inline packages/server/src/seed/seed.controller.ts:157]: DR LI IN THE HOUSE

**Review by dfarooq610 — APPROVED** [review]

File: packages/common/index.ts [hunk 2]
```diff
@@ -317,6 +317,9 @@ export class KhouryDataParams {
   @IsInt()
   campus!: string;
 
+  @IsInt()
+  professor!: string;
+
```
Comment (willstenzel) [inline packages/common/index.ts:322]: I'm not exactly sure how validation works shouldn't the `IsInt` decorator match up with a `number` type?

**Review by willstenzel — CHANGES_REQUESTED** [review]
Would you be able to add test to show that a new professor is created when the professor flag is present?

File: packages/common/index.ts (hunk 2 above)
Comment (NEUDitao) [reply packages/common/index.ts:322]: I'm using line 318 as a precedent here -- it also uses `@IsInt()` on a string.

**Review by willstenzel — APPROVED** [review]

**Pull request merged**